tools/esp8266/html/serial.html

```diff
@@ -30,9 +30,9 @@ <h3>handle next buttons with care - test / debug only!!</h3>
                 <br/>
                 <input type="button" value="power limit 100%" class="btn" id="pwrlim2"/>
                 <input type="button" value="power limit 10%" class="btn" id="pwrlim"/>
+                <input type="button" value="Restart" class="btn" id="restart"/>
                 <input type="button" value="Turn Off" class="btn" id="turnoff"/>
                 <input type="button" value="Turn On" class="btn" id="turnon"/>
-                <input type="button" value="Clean &amp; Reboot" class="btn" id="clnstate"/>
```

Comment:
ClearState is a better label and should be retained imho.

Reply:
Da CleanState nicht funktioniert und Restart sehr viel sinnvoller zum Zurücksetzen aller Werte im RAM des Inverters ist, ist dieses stattdessen reingekommen. Fehlte ja genau genommen auch.

Reply:
Select inverter Anzahl sollte noch dynamisch werden

Reply:
Check https://github.com/lumapu/ahoy/pull/330 is finished now.